presidio-analyzer/presidio_analyzer/nlp_engine/spacy_nlp_engine.py

```diff
@@ -59,7 +60,7 @@ def load(self) -> None:
 
     @staticmethod
     def _download_spacy_model_if_needed(model_name: str) -> None:
-        if not spacy.util.is_package(model_name):
+        if not (spacy.util.is_package(model_name) or Path(name).exists()):
```

Comment:
Should it be `name` or `model_name`?

Reply:
Ah sorry! I fat fingered this change. Fixed!